packages/active-campaign-client/src/__tests__/handlers/manageListSubscription.test.ts

```diff
@@ -0,0 +1,26 @@
+import { manageListSubscription } from '../../handlers/manageListSubscription';
+
+// remove .skip to run the test, be aware it does real API calls
+describe('manage list subscription', () => {
+  const cognitoUserId = '66ae52a0-f051-7080-04a1-465b3a4f44cc';
+  const listName = 'test-webinar-1732097286071';
```

Comment:
I will convert this 2 const in env variable or  add a TODO to refactor this test later